Description: Fix test relying on faulty behavior
 assert.throws did not test for the error message so far.
 This changes it to actually test for the error message.
Origin: upstream, https://github.com/jashkenas/coffeescript/commit/be2b243
Author: Ruben Bridgewater <ruben@bridgewater.de>
Forwarded: yes
Bug: https://github.com/jashkenas/coffeescript/issues/5028
Bug-Debian: https://bugs.debian.org/918491
Last-Update: 2019-01-08
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/exception_handling.coffee
+++ b/test/exception_handling.coffee
@@ -8,7 +8,7 @@
 # Throw
 
 test "basic exception throwing", ->
-  throws (-> throw 'error'), 'error'
+  throws (-> throw 'error'), /^error$/
 
 
 # Empty Try/Catch/Finally
